src/kagglehub/config.py

```diff
@@ -61,11 +61,7 @@ def get_kaggle_api_endpoint() -> str:
 
 def get_kaggle_credentials() -> Optional[KaggleApiCredentials]:
     # Check for credentials in the global variable
-    if (
-        _kaggle_credentials is not None
-        and _kaggle_credentials.username is not None
-        and _kaggle_credentials.key is not None
-    ):
+    if _kaggle_credentials:
```

Comment:
Can you add a test inside the `test_config.py` that test the following:

1. Call set_kaggle_credentials to set valid credentials
1. Call get_kaggle_credentials() & assert username & key
1. Call clear_kaggle_credentials()
1. Call get_kaggle_credentials() & assert value is None.